index.md

```diff
@@ -1,9 +1,23 @@
 ---
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
 layout: default
 ---
 Hello & welcome to Scanly, a university project for module 206CDE. The project is an open-source chatbot, licensed under the [GNU Lesser General Public Licence](/licence). For the task, we have to use [botkit.ai](https://botkit.ai/) & as BSc Cybersecurity & Ethical Hacking students, we have decided to make a chatbox frontend for security tools. The main aims that we have for this project is to make security tools more easier to use for beginers, & hope that this chatbox can be utilised by SMB IT Departments, in order to improve their network security.
 
-We only started the project W/c Monday 2nd October 2020 - so our in the 'live' development phase, and we hope to share more details with you in the near future, so watch this space!
\ No newline at end of file
+We only started the project W/c Monday 2nd October 2020 - so our in the 'live' development phase, and we hope to share more details with you in the near future, so watch this space!
+
+## Legal Policy & Terms of Use
+By using this software, you agree that you have permission from the network owners, or an appropriate authority, to run such assessment tools. 
+Unauthorised use may not be compliant with local laws.
+Scanly by DuckDuckGroup is open-source and as such we are not responsible for any possible issues.
+We accept NO responsibility for any legal consequences you may face.
```

Comment:
```suggestion
We accept NO responsibility for any legal consequences you may face. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```